third_party/move/move-compiler-v2/src/seqs_in_binop_checker.rs

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Aptos Foundation
```

Comment:
Please move into `env_pipeline`. All the other checkers on top-level should also go there, but this is for a future PR.

Reply:
done